docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,49 +1,54 @@
 
 ## General
-
-- [ ] Are the tests passing locally and on Travis?
-- [ ] Is the documentation up to date?
-
-## Components
-- [ ] Has a storybook story been created or updated?
-- [ ] Is the component accessible? 
-  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
-  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
-  - [ ] Is it passing accessibility checks in the storybook?
-
-## Apps
-
-- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
-- [ ] Does it work on mobile?
-- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?
-
-## Publishing
-
-- [ ] Is the changelog updated?
-- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?
-
-## Post-merging
-
-- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
-- [ ] Is the new feature working or bug now fixed?
-  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
-- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
-  - [ ] Is this approved by our designer?
-- [ ] Is this ready for production deployment?
+- [ ] Tests are passing locally and on Github.
+- [ ] Documentation is up to date. Changelog has been updated if appropriate.
+- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected.
+- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari.
+- [ ] FEM works in a mobile browser.
+
+## General UX
+- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
+- [ ] Can submit a classification
+- [ ] Can sign-in and sign-out
+- Example Staging Projects
+    - [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
+    - [Transformers](https://local.zooniverse.org:3000/projects/darkeshard/test-project-2022)
+
+## Bug Fix
+- [ ] The PR creator has listed user actions to use when testing if bug is fixed.
+- [ ] The bug is fixed.
+- [ ] Unit tests are added or updated.
+
+## New Feature
+- [ ] The PR creator has listed user actions to use when testing the new feature.
+- [ ] Unit tests are included for the new feature.
+- [ ] A storybook story been created or updated.
+- [ ] The component is accessible.
+  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
+  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
+  - It is passing accessibility checks in the storybook.
```

Comment:
Accessibility should also be tested on bug fixes and refactors, to make sure that regressions haven't been introduced.